packages/formspree-react/src/ValidationError.tsx

```diff
@@ -1,27 +1,27 @@
-import React from 'react';
-import type { FormError } from '@formspree/core';
+import React, { type ComponentPropsWithoutRef } from 'react';
+import type { FieldValues, SubmissionErrorResult } from '@formspree/core';
 
-export type ValidationErrorProps = {
+export type ValidationErrorProps<T extends FieldValues = FieldValues> = {
+  errors: SubmissionErrorResult<T> | null;
+  field?: keyof T;
   prefix?: string;
-  field?: string;
-  errors: FormError[];
-  [x: string]: any;
-};
+} & ComponentPropsWithoutRef<'div'>;
 
 export const ValidationError: React.FC<ValidationErrorProps> = (props) => {
   const { prefix, field, errors, ...attrs } = props;
+  if (errors == null) {
+    return null;
+  }
 
-  const error = (errors || []).find((error) => {
-    return error.field === field;
-  });
-
-  if (!error) {
+  const errs = field ? errors.getFieldErrors(field) : errors.getFormErrors();
+  if (errs.length === 0) {
     return null;
   }
 
   return (
     <div {...attrs}>
-      {prefix} {error.message}
+      {prefix ? `${prefix} ` : ''}
+      {errs.map((err) => err.message).join(', ')}
```

Comment:
If we have multiple errors, we'll just join them together for now. In the future PR, we could consider allow passing `renderError` props for customizing how to render each error (i.e. wrapped in a `<p>` with some styles or using a custom component).